src/App.js

```diff
@@ -127,32 +127,14 @@ const render = ({
           </Switch>
           <Modal
             isOpen={!!modalState.component}
-            style={{
-              overlay: {
-                position: 'fixed',
-                top: '0px',
-                left: '0px',
-                right: '0px',
-                bottom: '0px',
-                backgroundColor: 'rgba(0, 0, 0, 0.5)',
-                display: 'block',
-                zIndex: '111',
-              },
-              content: {
-                position: 'initial',
-                border: '1px solid rgb(204, 204, 204)',
-                background: 'rgb(255, 255, 255)',
-                borderRadius: '4px',
-                margin: '30px auto',
-                width: '55%',
-                boxShadow: 'rgba(0, 0, 0, 0.5) 0px 5px 15px',
-                overflow: 'visible',
-              },
+            {...{
+              unsetModal: effects.unsetModal,
+              setModal: effects.setModal,
+              title: modalState.title,
+              content: modalState.component,
+              appElement,
```

Comment:
Is app element used? 
having the default value be a document selector is going to be slow, so if it's not used lets remove it. if it is, maybe we can refactor to not need it :D

Reply:
@Jephuff it's fed into react-modal I believe. I can have it wrapped in a service function and load it in the Modal component directly

Reply:
ahh I see. So doing that is still going to run that selector every time it renders. 
If you can get away with passing it the `.app` div instead of the `#root` div you can use a ref to get the element and pass it in.

Otherwise you can just run that selector once and store the dom node, it won't change (it could maybe go in `injectGlobals`